.github/workflows/jest-tests.yml

```diff
@@ -0,0 +1,32 @@
+name: Run Jest Tests
```

Comment:
I thought we had committed this change before? I can see the file here https://github.com/Tobi-Sheriff/movie-archive/blob/main/.github/workflows/jest-tests.yml

Reply:
Your branching is probably broken. Can you ensure all changes in the main branch are pulled into your branch first before adding your new changes? This will enable us wee the diff properly

Reply:
> I thought we had committed this change before? I can see the file here https://github.com/Tobi-Sheriff/movie-archive/blob/main/.github/workflows/jest-tests.yml

Yes, i didnt make any changes there, all did was to clean some of the uneccessary comments

Reply:
> Your branching is probably broken. Can you ensure all changes in the main branch are pulled into your branch first before adding your new changes? This will enable us see the diff properly

A won't that mean that i will loose all my current work / changes?

Reply:
The branch looks broken to me. You can do a soft reset back to the same head as master or the last common head your branch shares with master. Stash your changes, pull main into your branch, apply your stash, fix all merge conflicts. We shouldn't be seeing changes you already have in main in this branch